Snowflake.Data/Configuration/EasyLoggingLogLevel.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace Snowflake.Data.Configuration
+{
+    internal enum EasyLoggingLogLevel
+    {
+        OFF,
+        ERROR,
+        WARN,
+        INFO,
+        DEBUG,
+        TRACE
+    }
+
+    internal static class EasyLoggingLogLevelExtensions
+    {
+        public static EasyLoggingLogLevel From(string logLevel)
```

Comment:
should it be public when wrapping class is only internal?

Reply:
Public member of an internal class is effectively internal. Leaving it as public we emphasize that it is not an implementation detail, but rather something the class would like to expose to others.
You can check some discussions here: https://stackoverflow.com/questions/9302236/why-use-a-public-method-in-an-internal-class